pkg/vdr/key/creator.go

```diff
@@ -88,10 +97,82 @@ func (v *VDR) Create(keyManager kms.KeyManager, didDoc *did.Doc,
 	return &did.DocResolution{DIDDocument: createDoc(publicKey, keyAgr, didKey)}, nil
 }
 
+func getKeyTypeAndVerMethodType(kt kms.KeyType) (kms.KeyType, string) {
```

Comment:
I don't think you meant to return the `kms.KeyType`

Reply:
sure, I can use the key type directly

Reply:
the function now is simplified and renamed to `getVerMethodType`. It returns the VerificationMethod.Type value only